libsweep/protocol.c

```diff
@@ -55,14 +63,15 @@ static uint8_t sweep_protocol_checksum_response_scan_packet(sweep_protocol_respo
   return checksum % 255;
 }
 
-void sweep_protocol_write_command(sweep_serial_device_s serial, const uint8_t cmd[static 2], sweep_protocol_error_s* error) {
+void sweep_protocol_write_command(sweep_serial_device_s serial, const uint8_t cmd[2], sweep_protocol_error_s* error) {
   SWEEP_ASSERT(serial);
   SWEEP_ASSERT(cmd);
   SWEEP_ASSERT(error);
 
-  sweep_protocol_cmd_packet_s packet = {.cmdByte1 = cmd[0],    //
-                                        .cmdByte2 = cmd[1],    //
-                                        .cmdParamTerm = '\n'}; //
+  sweep_protocol_cmd_packet_s packet;
+  packet.cmdByte1 = cmd[0];    //
+  packet.cmdByte2 = cmd[1];    //
+  packet.cmdParamTerm = '\n';  //
```

Comment:
Here and below: feel free to remove the `//` - they were used because `clang-format` can be a bit ugly from time to time.